Dockerfile

```diff
@@ -17,8 +17,11 @@ ENV DOTNET_CLI_TELEMETRY_OPTOUT=1
 RUN dotnet publish Jellyfin.Server --configuration Release --output="/jellyfin" --self-contained --runtime linux-x64 "-p:GenerateDocumentationFile=false;DebugSymbols=false;DebugType=none"
 
 FROM jellyfin/ffmpeg:${FFMPEG_VERSION} as ffmpeg
+
 FROM mcr.microsoft.com/dotnet/core/runtime:${DOTNET_VERSION}
-# libfontconfig1 is required for Skia
```

Comment:
Looks like this line was removed and hasnt been added back. We should keep this comment just in case someone gets "smart"

Reply:
I hate these comments, because now we'll need another one saying "va-mesa-drivers is for VAAPI" and whatever else we add in. It's not needed.

Reply:
Well, mesa is more or less self explainatory given what we are using now. you need mesa drivers for transcode.

It's not so obvious for `libfontconfig1` why we include it.

I still say put it back given the "oddity" of adding `libfonconfig1`.

And yes, it should but it lets us know when we can remove it (if we ever can) vs us just leaving it in for all time.

Reply:
Re-added in a different format